src/builder.ts

```diff
@@ -4,6 +4,10 @@ import _defaults from './default-options';
 import * as lodash from 'lodash';
 import { getLogger } from './initializers/log4js';
 import orkaType from './orka-builder';
+import { AsyncLocalStorage } from 'async_hooks';
```

Comment:
❓ Will this break if used with node < 12.17 ?

Reply:
Yes this will break if node < 12.17

We could go for `v1.0.0`

Reply:
I could also enhance readme with the minimum NodeJS version